src/utils/sessionXCSpy.ts

```diff
@@ -1,5 +1,4 @@
 import xcomponentapi from "reactivexcomponent.js";
-import { xcLogLevel } from "reactivexcomponent.js";
 
 class SessionXCSpy {
```

Comment:
My comment is still open on this class

Prefer the use of get / set typescript definition
http://www.codingdefined.com/2015/05/get-and-set-property-in-typescript.html